input-stream/src/main/java/software/amazon/s3/analyticsaccelerator/io/physical/data/Blob.java

```diff
@@ -92,6 +102,15 @@ public int read(byte[] buf, int off, int len, long pos) throws IOException {
     Preconditions.checkArgument(0 <= len, "`len` must not be negative");
     Preconditions.checkArgument(off < buf.length, "`off` must be less than size of buffer");
 
+    LoggingUtil.LogBuilder logger =
```

Comment:
add a condition to log only if (len > 5).... Parquet has a quirk were you get thousands of 1 byte reads in this method, that we don't care about measuring and it makes your logs way too noisy and impossible to parse